pages/vi/profiles/sawhuang.md

```diff
@@ -0,0 +1,17 @@
+# Who is the man behind the curtain?
+
+## The puppet master behind the veil
+## Who compiles without fail
+## The artist who paints in lines
+## Who's work you may find online
+
+My name is **Sawyer Huang**, and I am a freshman at [Cornell University](https://www.cornell.edu/) studying [Computer Science](https://www.cs.cornell.edu/).
+I'm from a small town called Gardiner, NY, but I currently reside in Ithaca, NY as a full-time student.
+When I'm not programming, you can probably find me at the gym lifting weights.
+
+I am excited because I believe this internship is a great opportunity to gain exposure to modern professional coding practices.
+I want to serve as a reliable team member on all endeavors throughout my time at OLE.
+
+Location (hometown)      | Timezone      | OS  
+:------------- |:-------------:| -----:
```

Comment:
markdown use is good but alignment is causing things not in place. (Inconsistent layout)